.gitignore

```diff
@@ -1,4 +1,3 @@
-dist/
```

Comment:
We definitely want to keep the `dist` folder in gitignore. Can you please add it back?